wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -114,7 +115,8 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 
 #### Rules
 
--   **Basic hyperdashes (1/1, 1/2) may begin to appear, but must not be used in conjunction with each other.** Platters should serve as an introduction to hyperdashing, meaning strong hypers (e.g. 1.5x above the trigger distance) and hypers combined with antiflow patterns must not be used. 1/3 and 1/4 hyperdashes may be used singularly (not in conjunction with other dashes or hyperdashes) if the BPM is less than 120.
+-   **Basic hyperdashes (1/1, 1/2) may begin to appear, but must not be used in conjunction with each other.** Platters should serve as an introduction to hyperdashing, meaning strong hypers (e.g. 1.5x above the trigger distance) and hypers combined with antiflow patterns must not be used.
```

Comment:
Antiflow patterns? No definition present
You should mention that 1.5x refers to Distance Snap